eclipse-collections/src/main/java/org/eclipse/collections/impl/bag/mutable/MultiReaderHashBag.java

```diff
@@ -738,6 +746,13 @@ public MutableBooleanBag collectBoolean(BooleanFunction<? super T> booleanFuncti
             return this.getDelegate().collectBoolean(booleanFunction, target);
         }
 
+        @Override
```

Comment:
@goldbal330 All of these are ok in `UntouchableMutableBag`, but we are missing the equivalent methods with the corresponding try with resources checks in the outer `MultiReaderHashBag`.